labs.yml

```diff
@@ -10,14 +10,6 @@ min_python: 3.10
 commands:
   - name: analyze
     description: Analyze source code from existing technology
-    flags:
```

Comment:
IMO we should have both the options, people can pass it in command line if they forget we prompt them